python/pyspark/sql/connect/column.py

```diff
@@ -263,6 +263,22 @@ def __str__(self) -> str:
         return f"Column({self._unparsed_identifier})"
 
 
+class SQLExpression(Expression):
```

Comment:
I still don't like kind of naming .. but this is at least somewhat consistent with what we have in DSv2 so I am fine.

Reply:
Let's see in the future... I guess we will need to  name more...